--- wml_backend/p3_eperl/configure.ORIG	2008-01-20 21:49:33.603878058 +0200
+++ wml_backend/p3_eperl/configure	2008-01-22 22:23:34.600763859 +0200
@@ -678,7 +678,11 @@
 #define HAVE_PERL_DYNALOADER 1
 EOF
 
-        perl_dla=$perl_archlib/auto/DynaLoader/DynaLoader.a
+        if $perlprog -e 'exit(! ($] > 5.009))' ; then
+            perl_dla=
+        else
+            perl_dla="$perl_archlib/auto/DynaLoader/DynaLoader.a"
+        fi
         ;;
     * )
         rc=no
--- wml_backend/p3_eperl/configure.in.ORIG	2008-01-22 22:40:02.234154859 +0200
+++ wml_backend/p3_eperl/configure.in	2008-01-22 22:23:07.086740195 +0200
@@ -159,7 +159,12 @@
     define )
         rc=yes
         AC_DEFINE(HAVE_PERL_DYNALOADER)
-        perl_dla=$perl_archlib/auto/DynaLoader/DynaLoader.a
+        if $perlprog -e 'exit(! ($] > 5.009))' ; then
+            perl_dla=
+        else
+            perl_dla="$perl_archlib/auto/DynaLoader/DynaLoader.a"
+        fi
+
         ;;
     * )
         rc=no
