--- wml-2.0.9/wml_backend/p3_eperl/configure.perllibs	2003-12-14 21:59:52.000000000 +0100
+++ wml-2.0.9/wml_backend/p3_eperl/configure	2003-12-15 00:16:18.000000000 +0100
@@ -658,7 +658,7 @@
 
 echo $ac_n "checking for Perl library files""... $ac_c" 1>&6
 echo "configure:661: checking for Perl library files" >&5
-perl_libs="`$perl_prog -e 'use Config; print $Config{libs}'`"
+perl_libs="`$perl_prog -e 'use Config; print $Config{perllibs}'`"
 echo "$ac_t""$perl_libs" 1>&6
 
 
--- wml-2.0.9/wml_backend/p3_eperl/configure.in.perllibs	2002-10-18 00:17:41.000000000 +0200
+++ wml-2.0.9/wml_backend/p3_eperl/configure.in	2003-12-15 00:16:31.000000000 +0100
@@ -144,7 +144,7 @@
 AC_SUBST(perl_ldflags)
 
 AC_MSG_CHECKING([for Perl library files])
-perl_libs="`$perl_prog -e 'use Config; print $Config{libs}'`"
+perl_libs="`$perl_prog -e 'use Config; print $Config{perllibs}'`"
 AC_MSG_RESULT([$perl_libs])
 AC_SUBST(perl_libs)
 
