diff -Naur wml-2.0.11/config/configure.in wml-2.0.11.oden/config/configure.in
--- wml-2.0.11/config/configure.in	2005-12-01 18:50:13.000000000 +0100
+++ wml-2.0.11.oden/config/configure.in	2007-09-28 14:41:33.000000000 +0200
@@ -116,7 +116,7 @@
 else
     AC_CHECK_PERL_MODULE(Getopt::Long, 2.16, getoptlong)
     AC_CHECK_PERL_MODULE(Bit::Vector, 5.2, bitvector)
-    AC_CHECK_PERL_MODULE(File::PathConvert, ANY, filepathconvert)
+	AC_CHECK_PERL_MODULE(File::Spec, ANY, filespec)
     AC_CHECK_PERL_MODULE(Image::Size, 2.6, imagesize)
     AC_CHECK_PERL_MODULE(IO::File, 1.07, io)
     AC_CHECK_PERL_MODULE(Term::ReadKey, 2.11, termreadkey)
diff -Naur wml-2.0.11/configure wml-2.0.11.oden/configure
--- wml-2.0.11/configure	2007-09-28 14:42:48.000000000 +0200
+++ wml-2.0.11.oden/configure	2007-09-28 14:41:33.000000000 +0200
@@ -275,7 +275,7 @@
 ac_unique_file="INSTALL"
 ac_default_prefix=/usr/local
 ac_subdirs_all="$ac_subdirs_all wml_backend/p2_mp4h wml_backend/p3_eperl wml_backend/p4_gm4 wml_aux/iselect wml_common/gd"
-ac_subst_vars='SHELL PATH_SEPARATOR PACKAGE_NAME PACKAGE_TARNAME PACKAGE_VERSION PACKAGE_STRING PACKAGE_BUGREPORT exec_prefix prefix program_transform_name bindir sbindir libexecdir datadir sysconfdir sharedstatedir localstatedir libdir includedir oldincludedir infodir mandir build_alias host_alias target_alias DEFS ECHO_C ECHO_N ECHO_T LIBS WML_VERSION VERSION WML_CONFIG_ARGS MAINT built_system built_user built_date libsubdir tmpdir build build_cpu build_vendor build_os host host_cpu host_vendor host_os target target_cpu target_vendor target_os DCOLON CC CFLAGS LDFLAGS CPPFLAGS ac_ct_CC EXEEXT OBJEXT CPP RANLIB ac_ct_RANLIB SET_MAKE PATH_PERL perlprog perlvers PERL_OPTS have_getoptlong have_bitvector have_filepathconvert have_imagesize have_io have_termreadkey INSTALLPRIVLIB INSTALLARCHLIB catdir subdirs LIBOBJS LTLIBOBJS'
+ac_subst_vars='SHELL PATH_SEPARATOR PACKAGE_NAME PACKAGE_TARNAME PACKAGE_VERSION PACKAGE_STRING PACKAGE_BUGREPORT exec_prefix prefix program_transform_name bindir sbindir libexecdir datadir sysconfdir sharedstatedir localstatedir libdir includedir oldincludedir infodir mandir build_alias host_alias target_alias DEFS ECHO_C ECHO_N ECHO_T LIBS WML_VERSION VERSION WML_CONFIG_ARGS MAINT built_system built_user built_date libsubdir tmpdir build build_cpu build_vendor build_os host host_cpu host_vendor host_os target target_cpu target_vendor target_os DCOLON CC CFLAGS LDFLAGS CPPFLAGS ac_ct_CC EXEEXT OBJEXT CPP RANLIB ac_ct_RANLIB SET_MAKE PATH_PERL perlprog perlvers PERL_OPTS have_getoptlong have_bitvector have_filespec have_imagesize have_io have_termreadkey INSTALLPRIVLIB INSTALLARCHLIB catdir subdirs LIBOBJS LTLIBOBJS'
 ac_subst_files=''
 
 # Initialize some variables set by options.
@@ -2848,7 +2848,7 @@
 if test ".$with_openworld" = .no; then
     have_getoptlong=0
     have_bitvector=0
-    have_filepathconvert=0
+    have_filespec=0
     have_imagesize=0
     have_io=0
     have_termreadkey=0
@@ -2882,15 +2882,15 @@
 fi
 
 
-    echo "$as_me:$LINENO: checking for Perl module File::PathConvert (ANY)" >&5
-echo $ECHO_N "checking for Perl module File::PathConvert (ANY)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {use File::PathConvert }; print "ok" unless $@'`;
+    echo "$as_me:$LINENO: checking for Perl module File::Spec (ANY)" >&5
+echo $ECHO_N "checking for Perl module File::Spec (ANY)... $ECHO_C" >&6
+rc=`$path_perl -e 'eval {use File::Spec }; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
-    have_filepathconvert=1
+    have_filespec=1
     echo "$as_me:$LINENO: result: found" >&5
 echo "${ECHO_T}found" >&6
 else
-    have_filepathconvert=0
+    have_filespec=0
     echo "$as_me:$LINENO: result: not found" >&5
 echo "${ECHO_T}not found" >&6
 fi
@@ -3696,7 +3696,7 @@
 s,@PERL_OPTS@,$PERL_OPTS,;t t
 s,@have_getoptlong@,$have_getoptlong,;t t
 s,@have_bitvector@,$have_bitvector,;t t
-s,@have_filepathconvert@,$have_filepathconvert,;t t
+s,@have_filespec@,$have_filespec,;t t
 s,@have_imagesize@,$have_imagesize,;t t
 s,@have_io@,$have_io,;t t
 s,@have_termreadkey@,$have_termreadkey,;t t
diff -Naur wml-2.0.11/wml_common/Makefile.in wml-2.0.11.oden/wml_common/Makefile.in
--- wml-2.0.11/wml_common/Makefile.in	2005-12-01 18:50:13.000000000 +0100
+++ wml-2.0.11.oden/wml_common/Makefile.in	2007-09-28 14:42:15.000000000 +0200
@@ -37,7 +37,7 @@
 INITVARS = \
      have_getoptlong=@have_getoptlong@ \
      have_bitvector=@have_bitvector@ \
-     have_filepathconvert=@have_filepathconvert@ \
+     have_filespec=@have_filespec@ \
      have_imagesize=@have_imagesize@ \
      have_io=@have_io@ \
      have_termreadkey=@have_termreadkey@ \
diff -Naur wml-2.0.11/wml_frontend/wml.src wml-2.0.11.oden/wml_frontend/wml.src
--- wml-2.0.11/wml_frontend/wml.src	2005-12-01 18:50:13.000000000 +0100
+++ wml-2.0.11.oden/wml_frontend/wml.src	2007-09-28 14:41:33.000000000 +0200
@@ -49,7 +49,7 @@
 }
 
 use Getopt::Long 2.13;
-use File::PathConvert;
+use File::Spec;
 use IO::File 1.06;
 use Term::ReadKey;
 use Cwd;
@@ -338,7 +338,7 @@
             @opt_I_NEW = @opt_I_OLD;
 
             #   adjust -D options
-            $reldir = File::PathConvert::abs2rel("$dir", "$src");
+            $reldir = File::Spec->abs2rel("$dir", "$src");
             $reldir = "." if $reldir eq '';
             foreach $d (@opt_D) {
                 if ($d =~ m|^([A-Za-z0-9_]+)~(.+)$|) {
@@ -363,7 +363,7 @@
             }
 
             #   adjust -I options
-            $reldir = File::PathConvert::abs2rel("$dir");
+            $reldir = File::Spec->abs2rel("$dir");
             $reldir = "." if $reldir eq '';
             foreach $path (@opt_I) {
                 if ($path !~ m|^/|) {
@@ -477,7 +477,7 @@
     $reldir = $src;
     $reldir =~ s,(:?/|^)[^/]+$,,;
     ($cwd = Cwd::cwd) =~ s|/$||;
-    $reldir = File::PathConvert::abs2rel($cwd, "$cwd/$reldir");
+    $reldir = File::Spec->abs2rel($cwd, "$cwd/$reldir");
     $reldir = "." if $reldir eq '';
 }
 else {
@@ -958,7 +958,7 @@
     if ($src =~ m|/|) {
         $src_dirname = $src;
         $src_dirname =~ s|/+[^/]*$||;
-        $src_dirname = File::PathConvert::realpath("$src_dirname");
+        $src_dirname = Cwd::abs_path("$src_dirname");
     }
     else {
         $src_dirname  = $cwd;
