--- wml-2.0.9/config/aclocal.m4.require	2002-08-06 23:51:34.000000000 +0200
+++ wml-2.0.9/config/aclocal.m4	2003-12-14 21:21:20.000000000 +0100
@@ -67,7 +67,7 @@
 
 AC_DEFUN(AC_CHECK_PERL_MODULE,[dnl
 AC_MSG_CHECKING([for Perl module $1 ($2)])
-rc=`$path_perl -e 'eval {require $1 ifelse($2, ANY, , $2)}; print "ok" unless [$]@'`; 
+rc=`$path_perl -e 'eval {use $1 ifelse($2, ANY, , $2)}; print "ok" unless [$]@'`; 
 if test ".$rc" = .ok; then
     have_$3=1
     AC_MSG_RESULT([found])
--- wml-2.0.9/configure.require	2002-10-18 23:44:21.000000000 +0200
+++ wml-2.0.9/configure	2003-12-14 21:20:42.000000000 +0100
@@ -2639,7 +2639,7 @@
 else
     echo "$as_me:$LINENO: checking for Perl module Getopt::Long (2.16)" >&5
 echo $ECHO_N "checking for Perl module Getopt::Long (2.16)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require Getopt::Long 2.16}; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use Getopt::Long 2.16}; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_getoptlong=1
     echo "$as_me:$LINENO: result: found" >&5
@@ -2653,7 +2653,7 @@
 
     echo "$as_me:$LINENO: checking for Perl module Bit::Vector (5.2)" >&5
 echo $ECHO_N "checking for Perl module Bit::Vector (5.2)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require Bit::Vector 5.2}; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use Bit::Vector 5.2}; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_bitvector=1
     echo "$as_me:$LINENO: result: found" >&5
@@ -2667,7 +2667,7 @@
 
     echo "$as_me:$LINENO: checking for Perl module File::PathConvert (ANY)" >&5
 echo $ECHO_N "checking for Perl module File::PathConvert (ANY)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require File::PathConvert }; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use File::PathConvert }; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_filepathconvert=1
     echo "$as_me:$LINENO: result: found" >&5
@@ -2681,7 +2681,7 @@
 
     echo "$as_me:$LINENO: checking for Perl module Image::Size (2.6)" >&5
 echo $ECHO_N "checking for Perl module Image::Size (2.6)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require Image::Size 2.6}; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use Image::Size 2.6}; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_imagesize=1
     echo "$as_me:$LINENO: result: found" >&5
@@ -2695,7 +2695,7 @@
 
     echo "$as_me:$LINENO: checking for Perl module IO::File (1.07)" >&5
 echo $ECHO_N "checking for Perl module IO::File (1.07)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require IO::File 1.07}; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use IO::File 1.07}; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_io=1
     echo "$as_me:$LINENO: result: found" >&5
@@ -2709,7 +2709,7 @@
 
     echo "$as_me:$LINENO: checking for Perl module Term::ReadKey (2.11)" >&5
 echo $ECHO_N "checking for Perl module Term::ReadKey (2.11)... $ECHO_C" >&6
-rc=`$path_perl -e 'eval {require Term::ReadKey 2.11}; print "ok" unless $@'`;
+rc=`$path_perl -e 'eval {use Term::ReadKey 2.11}; print "ok" unless $@'`;
 if test ".$rc" = .ok; then
     have_termreadkey=1
     echo "$as_me:$LINENO: result: found" >&5
